client/src/index.css

```diff
@@ -1,13 +1,13 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    margin: 0;
```

Comment:
css?

also I'll add for all selectors box-sizing: border box and remove padding. Because in future it can cause a lt of UI bugs

Reply:
add -
padding: 0;
box-sizing: border-box;